src/components/ui/WorkExperienceCard.stories.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+/* globals alert */
+
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import WorkExperienceCard from './WorkExperienceCard'
+
+const getEditingCardIdParentFunction = (cardId) => {
+  alert(`Editing Card ID: ${cardId}`)
+}
+
+storiesOf('Core Components|WorkExperienceCard Component', module)
```

Comment:
The definition for `Core Components` are the ones that's generic enough to be consumed for multiple situations. For instance, it we have a `Card` component without any specifics related to the domain/business. You can observe the ones located in this category follows that principle (at least we try to follow that as much as possible 😅). 

Also remove the `Component` word from the navigation, since it's kinda redundant and we expect the navigation to be short and simple :)

```suggestion
storiesOf('UI Components|WorkExperienceCard', module)
```

Reply:
done